src/math/gaussPointsAndWeights.m

```diff
@@ -16,48 +16,48 @@
 % along with ONSAS.  If not, see <https://www.gnu.org/licenses/>.
 %
 %Integration Gauss Points based on https://keisan.casio.com/exec/system/1329114617
-function [xIntPoints, wIntPoints] = gaussPointsAndWeights ( numGaussPoints )
-  if numGaussPoints == 1    
+function [xIntPoints, wIntPoints] = gaussPointsAndWeights ( numGaussPointsAeroForce )
```

Comment:
I suggest not changing this variable in this function since this function is used also for integrating the inertial forces... this is a general gauss integration function, not specific for aerodynamic analysis.

Reply:
It was replaced by accident while using Ctrl + F replacement, now should be fixed. Thanks for the review.